src/templates/Homepage.js

```diff
@@ -1,134 +1,183 @@
 import React from 'react'
 import { graphql } from 'gatsby'
+import { withPreview } from 'gatsby-source-prismic'
 import Layout from '../components/Layout'
 import SliceZone from '../components/SliceZone'
 
-const Homepage = ({ data }) => {
+export const HomepageTemplate = ({ data }) => {
```

Comment:
Do you need to export this function?

Reply:
Probably, I'll have to check :)

Reply:
Yes, because the withUnpublishedPreview hoc then imports it